tests/rest/admin/test_admin.py

```diff
@@ -63,6 +63,7 @@ def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
         self.other_user = self.register_user("user", "pass")
         self.other_user_token = self.login("user", "pass")
 
+    @unittest.override_config({"experimental_features": {"groups_enabled": True}})
```

Comment:
Do we really only have one test that uses groups?

Edit: ah, I see. The flag changes whether or not the servlets get registered; the other tests register the servlets themselves.